phpstan.neon.dist

```diff
@@ -32,12 +32,9 @@ parameters:
   ignoreErrors:
     - '#^Call to an undefined method Symfony\\Component\\Config\\Definition\\Builder\\NodeDefinition\:\:#'
     - '#^Unsafe usage of new static\(\)\.$#'
-    ## Relative to Attribute PHP 8
-    - '#^Access to an undefined property Attribute\:\:\$id\.$#'
-    - '#^Call to an undefined method Attribute\:\:delete\(\)\.$#'
-    - '#^Call to an undefined static method Attribute\:\:checkAttributeQty\(\)\.$#'
-    - '#^Call to an undefined static method Attribute\:\:getAttributeMinimalQty\(\)\.$#'
-    - '#^Call to an undefined static method Attribute\:\:getAttributes\(\)\.$#'
+    - '#^Constructor of class PrestaShopBundle\\Service\\DataProvider\\Marketplace\\ApiClient has an unused parameter \$toolsAdapter\.$#'
+    - '#constructor expects Symfony\\Contracts\\Translation\\TranslatorInterface\|null, Symfony\\Component\\Translation\\TranslatorInterface given\.$#'
```

Comment:
Should we really have to ignore these Symfony issues?

Reply:
Yep, it's mostly related to deprecated methods/classes